docs/TESTING.md

```diff
@@ -2,25 +2,24 @@
 
 ## Requirements
 
-1. [Mocha - Test runner](https://mochajs.org/)
-2. [Chai - Assertions](http://chaijs.com/)
+1. [Jest - Test runner](https://facebook.github.io/jest/)
 
 ```javascript
-expect(foo).to.be.a('string');
-expect(foo).to.equal('bar');
-expect(foo).to.have.lengthOf(3);
-expect(tea).to.have.property('flavors').with.lengthOf(3);
+expect(foo).toBeInstanceOf('string');
+expect(foo).toBe('bar');
+expect(foo).toHaveLength(3);
+expect(tea).toHaveProperty('flavors');
 ```
 
-3. [Enzyme - React Component Testing](https://github.com/airbnb/enzyme)
+2. [Enzyme - React Component Testing](https://github.com/airbnb/enzyme)
 
 ## Running Tests
-`npm run test`
+`npm run test:dev`
```

Comment:
Should we do 'running specific tests'

npm run test:db
npm run test:server
npm run test:client